c_src/erocksdb.cc

```diff
@@ -1119,6 +1120,38 @@ erocksdb_close(
 
 }  // erocksdb_close
 
+ERL_NIF_TERM
+erocksdb_flush(ErlNifEnv* env, int argc, const ERL_NIF_TERM argv[])
+{
+
+    erocksdb::DbObject * db_ptr;
+    ERL_NIF_TERM ret_term;
+
+    ret_term=erocksdb::ATOM_OK;
+    db_ptr=erocksdb::DbObject::RetrieveDbObject(env, argv[0]);
+
+    if (NULL==db_ptr)
+    {
+        return enif_make_badarg(env);
+    }
+
+    if (db_ptr->m_Db == NULL)
+    {
+        return error_einval(env);
+    }
+
+    rocksdb::FlushOptions opts;
+    opts.wait = true;
+    rocksdb::Status status = db_ptr->m_Db->Flush(opts);
+
+    if (!status.ok())
+    {
+        return error_tuple(env, erocksdb::ATOM_ERROR_DB_REPAIR, status);
```

Comment:
why not add and use the flush specific error code rather than using the  ` erocksdb::ATOM_ERROR_DB_REPAIR`?